mmdet/datasets/pipelines/transforms.py

```diff
@@ -2022,8 +2022,8 @@ def get_indexes(self, dataset):
             list: indexes.
         """
 
-        indexs = [random.randint(0, len(dataset)) for _ in range(3)]
-        return indexs
+        indices = [random.randint(0, len(dataset)) for _ in range(3)]
```

Comment:
indexes or indices? seems using indexes

Reply:
`indexes` will be better here, because the function name is `get_indexes`. I will fix it.